metron-deployment/roles/yaf/tasks/yaf.yml

```diff
@@ -48,7 +48,3 @@
 
 - name: Install init.d service script
   template: src=yaf dest=/etc/init.d/yaf mode=0755
-
```

Comment:
I removed it based on a comment in this PR from @dlyle65535 .  Happy to put it back in if it makes sense.